README.md

```diff
@@ -38,6 +38,21 @@ autoPreprocess({
 })
 ```
 
+---
+
+Alteratively, the library can be used from a CDN, such as unpkg, and then the components will be registered as custom elements. This is especially useful for quick prototypes that do not need all the features provided.
```

Comment:
*Alternatively
Also perhaps put `unpkg` in backticks?

Reply:
unpkg is not code to put it in backticks. maybe a link can be more useful, but should it be to [unpkg.com/attractions](https://unpkg.com/attractions) or just [unpkg.com](https://unpkg.com)?

Reply:
It's not code, but neither is it a word. Let's have it be a link to unpkg.com/attractions